make/photon/clair-adapter/builder

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set +e
+
+if [ -z $1 ]; then
+  error "Please set the 'version' variable"
+  exit 1
+fi
+
+VERSION="$1"
+
+set -e
+
+# the temp folder to store binary file...
+mkdir -p binary
+rm -rf binary/harbor-scanner-clair || true
+
+cd `dirname $0`
+cur=$PWD
+
+# the temp folder to store distribution source code...
+TEMP=`mktemp -d ${TMPDIR-/tmp}/clair-adapter.XXXXXX`
+git clone https://github.com/danielpacak/harbor-scanner-clair.git $TEMP
+cd $TEMP; git checkout $VERSION; cd -
+
+echo 'build the clair adapter binary bases on the golang:1.12...'
```

Comment:
no const value defined for golang version?

Reply:
changed, the `golang:1.12.5` was used in builder stage.